src/testRunner/unittests/tsbuild/helpers.ts

```diff
@@ -461,34 +499,21 @@ interface Symbol {
         modifyFs: (fs: vfs.FileSystem) => void;
         subScenario: string;
         commandLineArgs?: readonly string[];
-        cleanBuildDiscrepancies?: () => ESMap<string, CleanBuildDescrepancy>;
+        discrepancyExplanation?: () => readonly string[];

```

Comment:
Maybe a comment like "An array of lines to be printed in order when a discrepancy is detected"?